provider/helm3/implementer.go

```diff
@@ -19,7 +19,9 @@ import (
 // * update to latest chart package
 // * udpate the paramateres for the function
 
-const DefaultUpdateTimeout = 300
+// #595 - DefaultUpdateTimeout is in ns
+// Per https://pkg.go.dev/helm.sh/helm/v3/pkg/action#Upgrade
+const DefaultUpdateTimeout = 300000000000
```

Comment:
Hi, could you please declare it as a `3 * time.Minute` instead? It's a time.Duration time https://github.com/helm/helm/blob/main/pkg/action/upgrade.go#L66

Reply:
Wow, talk about reverting to a junior dev there. I'll make that change. You'd like 3 minutes instead of 5? The original `300s` would be `5 * time.Minute`, which I believe is an appropriate default timeout for a Helm upgrade.

Reply:
5 minutes is also good

Reply:
Done.